src/drive/store/persistedState.js

```diff
@@ -13,7 +13,7 @@ const migrateSettings = async prevState => {
     analytics,
     wifiOnly
   } = prevState.mobile.settings
-  const { revoked } = prevState.mobile.authorization
+  const { revoked } = prevState.mobile.authorization || {}
```

Comment:
It is a smell to not have consistent value in a state. It is better to initialize the state with an empty value or `undefined` (aka a value that means no value in the type you use). It will be easier to anticipate possible values in the state, even with a non-hard-typed language.